bbc/simorgh
# Intergrate image placeholder

Resolves #449 

_This PR integrates the the placeholder image created in https://github.com/BBC-News/simorgh/pull/668_


## Test instructions 
To see the placeholder use browser tools to throttle connection. This need to be tested cross browser and with images of various ratios. 

**Example URL's**
16x9 landscape image on http://localhost:7080/news/articles/c0000000025o
Square-ish image on http://localhost:7080/news/articles/c0000000004o
Smaller than 16x9 landscape image on http://localhost:7080/news/articles/c0000000012o
Larger then 9x16 portrait image on http://localhost:7080/news/articles/c0000000002o

## Screenshot 
![image](https://user-images.githubusercontent.com/7791726/46096173-648d7180-c1b6-11e8-85ca-d825c2fc98b2.png)
